backend/src/apiserver/common/util_test.go

```diff
@@ -1,104 +0,0 @@
-// Copyright 2019 The Kubeflow Authors
```

Comment:
I don't see a replacement for this file. I think we should keep the tests.

Reply:
These functions were moved to `server` package